concepticondata/conceptlists/Miklashevsky-2018-506.tsv

```diff
@@ -0,0 +1,507 @@
+Miklashevsky-2018-506-480	480	шлепанец	shlepanets	flip flop	Clothes	concrete	8	m	0.6	3.69	2.63	1.62	1.66	4.88	6.3	3.66	2.03	5.07	3.1		
+Miklashevsky-2018-506-481	481	шлюпка	shlyupka	ship's boat	Transport	concrete	6	f	0.8	3.83	2.48	1.59	1.03	3.21	6.11	5.17	2.09	7.48	2.32		
+Miklashevsky-2018-506-482	482	шляпа	shlyapa	hat	Clothes	concrete	5	f	1.5	4.1	1.41	2.07	1.52	4.38	6.59	5.8	4.3	4.23	3.24	771	HAT
+Miklashevsky-2018-506-483	483	шмель	shmel	bumblebee	Animal	concrete	5	m	0.7	4.69	5	1.83	1.31	3.41	6.56	2.72	4.22	4.41	3.07		
+Miklashevsky-2018-506-484	484	шнурок	shnurok	shoelace	Clothes	concrete	6	m	0.9	3.79	1.55	1.48	1.52	4.31	6.48	6.31	2.12	4.4	3.38		
+Miklashevsky-2018-506-485	485	шоссе	shosse	highway	Ground	concrete	5	n	1.6	4.31	5.24	2.31	1.34	2.31	6.44	2.31	2.22	6.33	3.36		
+Miklashevsky-2018-506-486	486	шпала	shpala	sleeper	Ground	concrete	5	f	0.7	4.38	2	3.24	1.36	3.36	6.07	4.06	1.53	6.97	2.83		
+Miklashevsky-2018-506-487	487	шпатель	shpatel	palette-knife	Tool_power_grip	concrete	7	m	0.2	3.73	2.47	2.1	1.79	3.79	5.15	5.97	3.08	8.28	2.11		
+Miklashevsky-2018-506-488	488	шпингалет	shpingalet	catch	Tool_precise_grip	concrete	9	m	0.3	4.19	3.5	1.27	1.41	3.42	5.83	6.1	3.15	7.82	2.58		
+Miklashevsky-2018-506-489	489	шприц	shprits	syringe	Tool_precise_grip	concrete	5	m	0.9	3.72	1.55	2.24	1.52	4.31	6.59	5.43	2.59	5.53	2.48		
+Miklashevsky-2018-506-490	490	штиль	shtil	calm	Space	concrete	5	m	0.5	2.76	3.69	2.1	1.17	3.34	5.15	1.93	3.97	8.88	2.29		
+Miklashevsky-2018-506-491	491	штопор	shtopor	corkscrew	Tool_power_grip	concrete	6	m	0.5	4.84	2.89	2.21	2.68	4.79	6.64	6.23	2.65	8	3.6		
+Miklashevsky-2018-506-492	492	шум	shum	noise	Sound	concrete	3	m	1.8	2	6.17	1.66	1.21	1.62	3.97	2.4	4.33	4.67	4.97	1182	NOISE
+Miklashevsky-2018-506-493	493	шуруп	shurup	screw	Tool_precise_grip	concrete	5	m	0.5	4.23	1.79	1.62	1.18	4.33	6.52	5.86	3	6.12	3.33	1802	SCREW
+Miklashevsky-2018-506-494	494	шхуна	shhuna	schooner	Transport	concrete	5	f	0.4	3.4	2.83	1.41	1.36	2.57	3.85	4.8	3.28	9.7	1.52		
+Miklashevsky-2018-506-495	495	щеколда	schekolda	latch	Tool_precise_grip	concrete	7	f	0.4	3.28	3.5	1.45	1.41	4.92	5.53	5.9	2.56	7.17	2.8		
+Miklashevsky-2018-506-496	496	щетка	schetka	brush	Tool_precise_grip	concrete	5	f	1	4.84	2.57	1.76	2.18	4.46	6.61	6.1	2.59	4.2	4.93	1821	BRUSH
+Miklashevsky-2018-506-497	497	щипцы	schiptsyi	tongs	Tool_power_grip	concrete	5	plur_tant	0.4	3.9	2.14	1.55	1.1	5.24	6.07	5.9	2.35	6.07	3.25	272	TONGS
+Miklashevsky-2018-506-498	498	эйфория	eyforiya	euphoria	Sense_Emotion	abstract	7	f	0.7	2.2	3.52	2.63	2.47	1.66	4.07	1.97	5.59	11.78	2.9		
+Miklashevsky-2018-506-499	499	электричка	elektrichka	electric train	Transport	concrete	10	f	1.2	4.52	5.48	2.38	1.45	3.03	6.52	4.62	2.78	6.15	3.46		
+Miklashevsky-2018-506-500	500	эмоция	emotsiya	emotion	Sense_Emotion	abstract	6	f	1.5	3.91	3.96	2.69	2.64	2.39	4.07	3.03	4.76	7.23	5.7	2573	FEELING (EMOTION)
```

Comment:
Oh I see there is another candidate for FEELING (EMOTION). They are clear synonyms, I don't think it matters which one of them will be mapped to this concept set.

Reply:
I changed the one above.